src/ol/geom/geometry.js

```diff
@@ -202,17 +202,18 @@ ol.geom.Geometry.prototype.applyTransform = goog.abstractMethod;
  * @param {ol.Extent} extent Extent.
  * @return {boolean} `true` if the geometry and the extent intersect.
  * @function
- * @api
+ * @api stable
```

Comment:
As discussed we shouldn't export abstract methods. This all means repeating the doc blocks on the child methods instead of using `@inheritDoc`.